protoarray/src/main/java/tech/pegasys/teku/protoarray/ForkChoiceStrategy.java

```diff
@@ -427,6 +436,7 @@ public void onExecutionPayloadResult(
           protoArray.markNodeValid(blockRoot);
           break;
         case INVALID:
+          LOG.warn("Payload for block root {} was invalid", blockRoot);
```

Comment:
mean to leave this in? invalid isn't super uncommon i thought...

Reply:
Invalid signatures and stuff like that aren't uncommon, but invalid payload are.  Geth prints a very big, scary multiline warning if it gets a block that is invalid when executed (as opposed to invalid PoW). I'm a little unsure but I think its worth leaving this in at least until we get some experience on merge testnets to see how annoying it is as I expect it will be very useful during interop testing.